demo/basic.js

```diff
@@ -4,16 +4,15 @@ const renderer = PIXI.autoDetectRenderer({ width: 800, height: 600 });
 let stage;
 let tilemap;
 
+let atlas;
+let button;
+
 document.body.appendChild(renderer.view);
 
-const loader = new PIXI.Loader();
+const loadAssets = async () => {
+    atlas = await PIXI.Assets.load('assets/atlas.json');
+    button = await PIXI.Assets.load('assets/button.png');
 
-loader.add('atlas', 'assets/atlas.json');
-loader.add('button', 'assets/button.png');
-loader.load(function onAssetsLoaded(loader, resources)
```

Comment:
I think with PixiJS v7 support we could convert this to a modern version using arrow functions and anonymous functions.